e2e/app-template/template/src/api/database-dump/content-types/database-dump/schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "kind": "collectionType",
+  "collectionName": "database-dumps",
+  "info": {
+    "displayName": "database-dump",
+    "singularName": "database-dump",
+    "pluralName": "database-dumps",
+    "description": "",
+    "name": "database-dump"
```

Comment:
Should we call the content-type maybe just "database" or "e2e" or something like that and then register methods such as `dump`, `seed` on it instead of limiting the scope to dumping?